### PR TITLE
Feature/selectingseats

### PR DESCRIPTION
**Pull request by skrillfer**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,22 +3,63 @@ import {
...
+import { setStateSeat } from './redux/stage/stage.actions';
+import { selectCurrentUser } from './redux/user/user.selectors';
+
+let socket;
+
+export class App extends React.Component{
+  constructor(props){
+    super(props);
+    socket = io.connect("http://52.70.18.213");
+    socket.emit('conectado',{message:'Ha funcionado Perron'});
```
Comment (bragonya) [inline src/App.js]: No hay que dejar cosas quemadas

File: src/components/croquis/croquis.component.jsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 import { ReactComponent as StageX } from '../../assets/stage.svg';
 
-import event_seats_structure from '../../assets/seat-structure';
 
-const Croquis = () =>{
-    const {sections} = event_seats_structure;
-    const [SL1,SL2,VIP1,VIP2,PR1,PR2,PR3,PR4,E1,E2,E3,E4] = sections;
+const Croquis = ({mainStage}) =>{
+    const {SL1,SL2,VIP1,VIP2,PF1,PF2,PF3,PF4,E1,E2,E3,E4} = mainStage;
+    console.log(SL1);
```
Comment (bragonya) [inline src/components/croquis/croquis.component.jsx]: Remover este console

**Review by kenny08gt — APPROVED** [review]

**Review by bragonya — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (skrillfer) [reply src/App.js]: commit d51f831d6e0733d9bfbc510810406704b652e5b5

File: src/components/croquis/croquis.component.jsx (hunk 2 above)
Comment (skrillfer) [reply src/components/croquis/croquis.component.jsx]: commit b8b83c9f87d5195359d08e0ea0c66226e59f68ad

**Review by bragonya — APPROVED** [review]

**Pull request merged**